### PR TITLE
[W5.8] [W11-A1] Wayne Neo

### PR DESCRIPTION
**Pull request by wzneo**
As part of LO, added Non-Functional Requirements

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -94,6 +94,12 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should retain data in the event of a crash
+6. Should be able to process queries with less than 2s of delay
+7. Should be intuitive and easy-to-use to novice users
```
Comment (okkhoy) [inline doc/DeveloperGuide.md:99]: Define intuitive/easy-to-use.

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -94,6 +94,12 @@ Use case ends.
...
+8. Should have new features developed every week
+9. Should be able to import and export to different AddressBooks
```
Comment (okkhoy) [inline doc/DeveloperGuide.md:101]: This can be considered more of a functional requirement.

**Comment by okkhoy** [comment]
@AzuraAiR 
NFR (and requirements in general) should be as precise as possible.
Avoid adjectives like `easy`, `fast`, `intuitive` etc.,
Instead, use precise statements like `response time should be 1 second`, `detailed help which shows all options should be present`

Some comments added. Please ack & close the PR after reading comments.

**Comment by wzneo** [comment]
Noted with thanks

**Pull request closed without merging**